packages/cardano-services/src/Asset/openApi.json

```diff
@@ -0,0 +1,109 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "UTxO",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/asset/get-asset": {
+      "post": {
+        "summary": "Get Asset Info",
+        "operationId": "getAssetInfo",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/GetAssetRequest"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "success operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/AssetInfo"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "invalid request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "AssetInfo": {
+        "type": "object",
+        "properties": {
+          "assetId": {
+            "type": "string"
+          },
+          "fingerprint": {
+            "type": "string"
+          },
+          "mintOrBurnCount": {
+            "type": "number"
+          },
+          "name": {
+            "type": "string"
+          },
+          "policyId": {
+            "type": "string"
+          }
+        }
+      },
+      "ExtraData": {
+        "type": "object",
+        "properties": {
+          "history": {
+            "type": "boolean"
+          },
+          "nftMetadata": {
+            "type": "boolean"
+          },
+          "tokenMetadata": {
+            "type": "boolean"
+          }
+        }
+      },
+      "GetAssetRequest": {
+        "required": ["args"],
+        "type": "object",
+        "properties": {
+          "args": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "oneOf": [
```

Comment:
Here I used `oneOf` in order to accept `[string, ExtraData]`, but this will also allow `[ExtraData, string]` (... or `[string, string]`).
Who could I change this in order accept only `[string, ExtraData]`?
Thank you

Reply:
You can leave this for now, as we'll be refactoring the providers to use a single object argument rather than depending on the position in the argument array.